llbuild.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 55;
+	objectVersion = 70;
```

Comment:
Should probably not go beyond 63 for now, as further versions are still in beta and therefore this may break builds.